src/styles/blocks/calendar.scss

```
@@ -0,0 +1,57 @@
+@import "../utils/_variables";
+@import "../utils/normalize";
+
+@mixin square ($side) {
+  height: $side;
+  width: $side;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-box-size;
+  gap: $box-gap;
+  padding: 20px 0;
+  margin: 0 auto;
+  align-items: center;
+
+  @for $i from 1 through 31 {
+    & :nth-child(#{$i})::before {
+      content: "#{$i}";
+    }
+  }
+  
+  @each $day, $dayNum in $days-of-week {
+    &--start-day--#{$day} :first-child {
+      margin-left: $dayNum * ($calendar-dey-box + $box-gap);
+    }
+  }
+  
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    position: relative;
+    @include square($calendar-dey-box);
+
+    background-color: #eee;
+    border: 1px solid black;
```

don't forget to create variables for all colors
```suggestion
    border: 1px solid $c-black;
```